### ISSUE TITLE
Can't even use this

### ISSUE THREAD

**Issue by KuntKisser** [termux/termux-packages#17489]
Everytime I try to put in a command, it says "CANNOT LINK EXECUTABLE "apt": cannot locate symbol "__emutls_get_address" referenced by "/data/data/com.termux/files/usr/lib/libapt-pkg.so"..."

**Comment by twaik** [comment]
After installing Termux and opening it, you should do the following to make sure you have it up-to-date:

**1.** `termux-change-repo` = Lets you select where to get updates from:
- `SPACE` toggles highlighted options;
- `ENTER` moves to the next screen;
Choose 1st topmost option on each screen _(if that one fails, try different combination)_
**2.** `pkg update && pkg upgrade` = First it updates our package database to see what can be upgraded, and then we actually upgrade all packages.
On each prompt press `y` and `Enter` to install newest version available
**3.** `exit` = This exits Termux, so you can restart it yourself - Termux will start up all ready to use, all updated!
_TL:DR: everything in one command with autoaccepting each update:_
```
termux-change-repo ; yes | pkg update && yes | pkg upgrade ; exit
```
